### DISCUSSION TITLE
How do I un-focus main URL?

### DISCUSSION THREAD

**Discussion by TheYellowArchitect** [philc/vimium#4390] [General]
If I open a new tab, the focus is on main URL, and my vim inputs are lost. I can spam ESC, nothing happens...
To reproduce this: Press <Ctrl+l> (L lower-case)

**Comment by rM-self-serve** [comment]
Is this firefox? On firefox I have success with F6.

**Reply by TheYellowArchitect** [reply]
Thank you. That indeed worked!
